front-end-client/src/pages/home/home.html

```diff
@@ -9,8 +9,44 @@
   </ion-navbar>
 </ion-header>
 
+
 <ion-content padding>
   <a ion-button full="true" color="primary" href="https://interfaithsanctuary.org/donate/">Online Cash Donation</a>
+
+  <ion-grid>
+    <ion-row>
+      <ion-item>
+        <ion-label>Callout</ion-label>
+        <ion-select [(ngModel)]="Callout">
+          <ion-option value="Temp1">Temp1</ion-option>
+          <ion-option value="Temp2">Temp2</ion-option>
+          <ion-option value="Temp3">Temp3</ion-option>
+        </ion-select>
+      </ion-item>
+    </ion-row>
+
+    <ion-row>
+      <ion-col col-9>
+        <ion-item>
+          <ion-label>Sort By</ion-label>
+          <ion-select [(ngModel)]="sortBy" interface=popover>
+            <ion-option value="priority">Priority</ion-option>
+            <ion-option value="chronological">Chronological</ion-option>
+            <ion-option value="fulfillment">Fulfillment</ion-option>
+          </ion-select>
+        </ion-item>
+      </ion-col>
+      
+      <ion-col>
+        <button ion-button
```

Comment:
What does this button do? It looks like the dropdown list already sorts the callouts, and that this button might sort them alphabetically. Could an alphabetical ordering be added to the dropdown? I could be wrong about the intentions of the button - if the button isn't redundant, it all looks good to me!

Reply:
The callout dropdown is so the user can select a specific callout if they would like, and the sort dropdown is to change the order of the needs currently shown. With a basic implementation of callouts, the priority and chronological sorts might not be as useful as say fulfillment, which would allow the user to sort needs based on fulfillment percentage.

Reply:
Cool! The purpose of the callout dropdown was pretty clear from the UI design, but I was wondering what the button with the arrow on it does! (the one that's hooked up to toggleOrdering())

Reply:
Oh! That will toggle the order between ascending and descending of the current sort type selected. The arrow was the best thing I could think of that would fit within the space provided, but I'm open to other suggestions. I'm also thinking that eventually that sort/filter area with be redesigned and refactored into its own component.